src/metabase/server.clj

```diff
@@ -23,12 +23,14 @@
 (defn- jetty-config []
   (cond-> (m/filter-vals
            some?
-           {:port          (config/config-int :mb-jetty-port)
-            :host          (config/config-str :mb-jetty-host)
-            :max-threads   (config/config-int :mb-jetty-maxthreads)
-            :min-threads   (config/config-int :mb-jetty-minthreads)
-            :max-queued    (config/config-int :mb-jetty-maxqueued)
-            :max-idle-time (config/config-int :mb-jetty-maxidletime)})
+           {:port                 (config/config-int :mb-jetty-port)
+            :host                 (config/config-str :mb-jetty-host)
+            :max-threads          (config/config-int :mb-jetty-maxthreads)
+            :min-threads          (config/config-int :mb-jetty-minthreads)
+            :max-queued           (config/config-int :mb-jetty-maxqueued)
+            :request-header-size  (config/config-int :mb-jetty-maxrequestheader)
+            :response-header-size (config/config-int :mb-jetty-maxresponseheader)
```

Comment:
Could we rather keep with the naming scheme of the other variables? That'll strengthen the Metabase options' references to the specific Jetty options.

 ```suggestion
             :request-header-size  (config/config-int :mb-jetty-requestheadersize)
             :response-header-size (config/config-int :mb-jetty-responseheadersize)
```

Reply:
Any reason to not follow our convention of not combining separate words into one mega-word?

This PR adds things like `MB_JETTY_REQUESTHEADERSIZE`, I personally think `MB_JETTY_REQUEST_HEADER_SIZE` is a bit more readable. Even better would be something like `MB_JETTY_REQUEST_HEADER_SIZE_BYTES` that makes it clear at a glance what unit to use

Reply:
@walterl I think in this case `MB_JETTY_MAX_REQUEST_HEADER_SIZE_BYTES` would be clearest. `:request-header-size` isn't a name used internally in Jetty. It's the name Ring uses for some Jetty setting somewhere. So just because Ring names the Jetty option `:request-header-size` doesn't mean we have to follow suit and use an ambiguous name for a Jetty option.

I'd rather use clear and unambiguous stuff even if it doesn't line up 100% with the param names used by Ring

Reply:
I'll similarly update the options around these new ones, once this PR is merged. :+1:

Reply:
Sorry for the delay. I agree with the discussed changes, made chagnges according to the discussions and have now committed them. 

This is my first push request ever, so if I'm completely honest, not sure what do I have to do now.